.github/workflows/api_refs.yaml

```
@@ -4,7 +4,7 @@ on:
     workflow_dispatch:
         inputs:
             version:
-                description: 'Released version'
+                description: 'Released version (for example: v4.6.21, v5.0.0)'
```

Let's detail as much as possible
```suggestion
                description: 'Released version tag (for example: v4.6.21 or v5.0.0)'
```